common/djangoapps/third_party_auth/pipeline.py

```diff
@@ -704,7 +704,7 @@ def set_logged_in_cookies(backend=None, user=None, strategy=None, auth_entry=Non
                     pass
                 else:
                     response = redirect(redirect_url)
-                    return user_authn_cookies.set_logged_in_cookies(request, response, user)
+                    user_authn_cookies.set_logged_in_cookies(request, response, user)
```

Comment:
is this a temporary solution? can you add a TODO here?

Reply:
done